pkg/apis/integreatly/v1alpha1/rhmi_types.go

```diff
@@ -68,6 +68,7 @@ var (
 	VersionRHSSO               ProductVersion = "7.4"
 	VersionRHSSOUser           ProductVersion = "7.4"
 	VersionMonitoringSpec      ProductVersion = "1.0"
+	VersionSolutionExplorer    ProductVersion = "master"
```

Comment:
This will be updated to an actual version of SolX when an updated version of the web-app operator is included.

Reply:
I tested this PR against the master version of integreatly-operator. The version of SolX is `2.25.1`
```json
{
  "name": "solution-explorer",
  "phase": "completed",
  "products": {
    "solution-explorer": {
      "host": "https://solution-explorer.apps.osd-4-4-7.f2a8.s1.devshift.org",
      "name": "solution-explorer",
      "operator": "0.0.57",
      "status": "completed",
      "version": "2.25.1"
    }
  }
}
```

Reply:
Yes, #842 was merged, so updating the solx version here and removing the hold on this PR now